app/konnect/dev-portal/developers/dev-gen-creds.md

```diff
@@ -23,7 +23,7 @@ A credential, or API key, generated in the {{site.konnect_short_name}} Dev Porta
    using your `key-auth` credential:
 
    ```
-   <proxy-url>/<service>/?apikey=<apikey>
+   <proxy-url or proxy-ip>/<route>?apikey=<apikey>
```

Comment:
```suggestion
   {PROXY_URL_OR_PROXY_IP}/{ROUTE}?apikey={APIKEY}
```
We're transitioning variables in the docs to this formatting.

 
I think it's also useful to add a real example here:

> For example, the final URL might look something like this:
> 
> ```
> http://localhost:8000/mock/?apikey=mykey
>```